partner/app/tests/test_app.py

```diff
@@ -94,6 +94,75 @@ def test_read_tilesp_validate_sub2(client, sub2):
     assert "response" in response_content
 
 
+@pytest.mark.parametrize(
+    "country_code",
+    [
+        "CA",
+        "DE",
+        "GB",
+        "FR",
+        "AU",
+        "IT",
+        "MX",
+        "IN",
+        "BR",
+        "ES",
+    ],
+    ids=[
```

Comment:
I don't believe that these are used. Is there a reason to keep these?

Reply:
Ah, this may be my confusion around `pytest.parameterize`.
In this case, the `ids` are not being passed in, but are labeling the values that are.

Reply:
Yeah, Ids are just labels. :)